submissions/EmmanuelIdiaghe/php-database/admin.php

```diff
@@ -0,0 +1,67 @@
+                echo "<tr>";
+                    echo "<th style= 'border: 1px solid black'>S/N</th>";
+                    echo "<th style= 'border: 1px solid black'>Full Name</th>";
+                    echo "<th style= 'border: 1px solid black'>Email Address</th>";
+                    echo "<th style= 'border: 1px solid black'>Created At</th>";
+                    echo "<th style= 'border: 1px solid black'>Actions</th>";
+                echo "</tr>";
+                foreach ($info as $value) {
+                    echo "<tr>";
+                        echo "<td style= 'border: 1px solid black'>" . $value["id"] . "</td>";
+                        echo "<td style= 'border: 1px solid black'>" . $value["first_name"] . "  " . $value["last_name"] . "</td>";
+                        echo "<td style= 'border: 1px solid black'>" . $value["email_address"] . "</td>";
+                        echo "<td style= 'border: 1px solid black'>" . $value["created_at"] . "</td>";
+                        echo "<td style= 'border: 1px solid black'><form method='get' action='view.php'><button style= 'color: black; background-color: #DAF943; border-radius: 20%; margin-top: 19px;' type='submit' name='view' value=\"$value[id]\">View</button></form></td>";
+                    echo "</tr>";
+                }
+            echo "</table";
+        } else echo "No record found!";
+    }
+    else {
+        $display = unserialize($_COOKIE["show"]);
+        if (count($display) > 0) {
+            echo "<table style= 'border: 1px solid black'>";
+                echo "<tr>";
+                    echo "<th style= 'border: 1px solid black'>S/N</th>";
+                    echo "<th style= 'border: 1px solid black'>Full Name</th>";
+                    echo "<th style= 'border: 1px solid black'>Email Address</th>";
+                    echo "<th style= 'border: 1px solid black'>Created At</th>";
+                    echo "<th style= 'border: 1px solid black'>Actions</th>";
+                echo "</tr>";
+                foreach ($display as $value) {
+                    echo "<tr>";
+                        echo "<td style= 'border: 1px solid black'>" . $value["id"] . "</td>";
+                        echo "<td style= 'border: 1px solid black'>" . $value["first_name"] . "  " . $value["last_name"] . "</td>";
+                        echo "<td style= 'border: 1px solid black'>" . $value["email_address"] . "</td>";
+                        echo "<td style= 'border: 1px solid black'>" . $value["created_at"] . "</td>";
+                        echo "<td style= 'border: 1px solid black'><form method='get' action='view.php'><button style= 'color: black; background-color: #DAF943; border-radius: 20%; margin-top: 19px;' type='submit' name='view' value=\"$value[id]\">View</button></form></td>";
+                    echo "</tr>";
+                }
+            echo "</table";
+        } else echo "No record found!";
+    }   
```

Comment:
You now the concept but there is code repetition here. Kindly look a better way to pass the data into the table without having to repeat yourself

Reply:
Sure, thanks!